backend/controllers/instance.server.controller.js

```diff
@@ -0,0 +1,59 @@
+let models = require("../models");
+
+exports.get = async (req, res) => {
+  if (req.params.id) {
```

Comment:
For whatever reason, this is always null and never is true

Reply:
Just checked the test the id is not null for the test, the issue was the findAll instead of findOne. PR to come...